packages/sdk/package.json

```diff
@@ -45,15 +46,17 @@
     "hardhat-deploy": "^0.11.4",
     "nyc": "^15.1.0",
     "typedoc": "^0.22.13",
-    "mocha": "^10.0.0"
+    "mocha": "^10.0.0",
+    "vitest": "^0.28.3"
   },
   "dependencies": {
     "@eth-optimism/contracts": "0.5.40",
     "@eth-optimism/core-utils": "0.12.0",
     "@eth-optimism/contracts-bedrock": "0.13.2",
     "lodash": "^4.17.21",
     "merkletreejs": "^0.2.27",
-    "rlp": "^2.2.7"
+    "rlp": "^2.2.7",
+    "zod": "^3.11.6"
```

Comment:
Does this need to be an actual dep or a dev dep?

Reply:
Right now it can get away with being a dev dep but I will almost certainly use it as a regular dep within the next month.   Every problem is a nail and zod is my hammer